process/block/interceptedBlocks/common.go

```diff
@@ -238,3 +245,16 @@ func checkMiniblocks(miniblocks []block.MiniBlockHeader, coordinator sharding.Co
 
 	return nil
 }
+
+func checkChainID(header data.HeaderHandler, chainID []byte) error {
```

Comment:
this function could be used in consensus as well. maybe the input parameters can be receivedChainID and ownChainID.

Reply:
Moved on headerHandler implementations. Changed in consensus and interceptors.